Cargo.toml

```diff
@@ -26,18 +26,21 @@ with-deprecated = []
 default = ["with-deprecated"]
 
 [dependencies]
+iovec    = "0.1.2"
 lazycell = "1"
 log      = "0.4"
+net2     = "0.2.33"
```

Comment:
Were the dependency version bumps necessary for adding Redox support?

Reply:
Yes, net2 only recently has redox support